metricbeat/module/elasticsearch/elasticsearch.go

```diff
@@ -288,13 +288,21 @@ func GetLicense(http *helper.HTTP, resetURI string) (*License, error) {
 }
 
 // GetClusterState returns cluster state information.
-func GetClusterState(http *helper.HTTP, resetURI string, metrics []string) (mapstr.M, error) {
+func GetClusterState(http *helper.HTTP, resetURI string, metrics []string, filterPaths []string) (mapstr.M, error) {
```

Comment:
Pretty sure this function is called from more places, since the signature changes, we should make sure that all calls are still compatible. Unless Go deals with missing parameters

Reply:
Yeah i was surprised as well 😅